anywhere-fitness/src/components/class-card.js

```diff
@@ -11,8 +54,14 @@ function ClassCard(props) {
       <p>Class Type: {props.details.type}</p>
       <p>Duration: {props.details.duration}</p>
       <p>IntensityLevel: {props.details.intensityLevel}</p>
-      <p>Current Attendees Number: {props.details.currentAttendeesNo}</p>
-      <p>Max Size: {props.details.maxsize}</p>
+      <p>Current Attendees Number: {props.details.currentRegistered}</p>
+      <p>Max Size: {props.details.maxClassSize}</p>
+      <p>Instuctor: {props.details.instructor}</p>
+
+      
+      {editing && <EditClasses details={props.details} saveEdit={saveEdit} />}
+      <button onClick={e => { addClass() }}>Add Class</button>
+      <button onClick={e => { removeClass() }}>Remove Class</button>
```

Comment:
You are not consuming the event. Also your handlers need no parameters passed in. Should be simplified to 
```
      <button onClick={addClass}>Add Class</button>

```